transform/snowflake-dbt/models/hightouch/blapi/blapi_cloud_account.sql

```diff
@@ -8,7 +8,8 @@
 WITH cloud_accounts_to_sync as (
     SELECT
         customers_with_cloud_subs.*,
-        '0053p0000064nt8AAA' AS ownerid
+        '0053p0000064nt8AAA' AS ownerid,
+        COALESCE(customers_with_cloud_subs.domain, '{{ var('unknown_account_name') }}' ) AS account_name
```

Comment:
still have mixed feelings about this, but I guess it got the approval 
of its consumers.

Reply:
Agree. We should spend some time to document the different conventions used, at least for the most common items.